docs/user/power/install_upi.md

```diff
@@ -0,0 +1,296 @@
+# Install: Power User Provided Infrastructure
+
+The steps for performing a UPI-based install are outlined here. Example automation is provided [here](#example-power-upi-configuration) to help model your own.
+
+## Table of contents
+
+1. [Minimum compute requirements](#minimum-resource-requirements)
+
+2. [Network topology requirements](#network-topology-requirements)
+
+3. [DNS requirements](#dns-requirements)
+
+4. [Getting Ignition configs for machines](#getting-ignition-configs-for-machines)
+
+5. [Booting machines with RHCOS and Ignition configs](#booting-machines-with-rhcos-and-ignition-configs)
+
+6. [Watching your installation (bootstrap complete, cluster available)](#watching-your-installation)
+
+7. [Example Bare-Metal UPI deployment](#example-power-upi-configuration)
+
+## Compute
+
+The smallest OpenShift 4.x clusters require the following host:
+
+* 1 bootstrap machine.
+
+* 3 control plane machines.
+
+* atleast 1 worker machine.
```

Comment:
nit "atleast" -> "at least".  Also, are you sure about this?  I think ingress needs two compute machines to schedule the router.

Reply:
This is from my sample cluster with one worker node. The default replicas is however 2 for highly available ingress router. 
```
oc describe clusteroperators/ingress
Name:         ingress
Namespace:
Labels:       <none>
Annotations:  <none>
API Version:  config.openshift.io/v1
Kind:         ClusterOperator
Metadata:
  Creation Timestamp:  2020-04-20T19:43:26Z
  Generation:          1
  Resource Version:    15151
  Self Link:           /apis/config.openshift.io/v1/clusteroperators/ingress
  UID:                 4c01e407-5b45-409d-a25e-e4a0a12b3637
Spec:
Status:
  Conditions:
    Last Transition Time:  2020-04-20T19:44:15Z
    Reason:                NoIngressControllersDegraded
    Status:                False
    Type:                  Degraded
    Last Transition Time:  2020-04-20T19:49:13Z
    Message:               desired and current number of IngressControllers are equal
    Status:                False
    Type:                  Progressing
    Last Transition Time:  2020-04-20T19:44:15Z
    Message:               desired and current number of IngressControllers are equal
    Status:                True
    Type:                  Available
  Extension:               <nil>
  Related Objects:
    Group:
    Name:       openshift-ingress-operator
    Resource:   namespaces
    Group:      operator.openshift.io
    Name:
    Namespace:  openshift-ingress-operator
    Resource:   IngressController
    Group:      ingress.operator.openshift.io
    Name:
    Namespace:  openshift-ingress-operator
    Resource:   DNSRecord
    Group:
    Name:       openshift-ingress
    Resource:   namespaces
  Versions:
    Name:     operator
    Version:  4.3.0-0.nightly-ppc64le-2020-02-20-212303
    Name:     ingress-controller
    Version:  quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:b3896981f985871cd7b286866f1987d885d10a52698f7117014b9ccdb26b8bd9
```